views/login/securityQuestion2.pug

```diff
@@ -9,16 +11,17 @@ block content
   h1 #{title}
 
   form.cra-form.cra-form--lg(method='post')
-    +radios('securityQuestion', 'Answer a question from the list below', null,[
-      {value: "/login/questions/bank", label: "Your bank or credit union account information"},
+    +radios('securityQuestion', 'Answer a question from this list to get your notice of assessment immediately after you file with this service:', null,[
       {value: "/login/questions/rrsp", label: "Details of your contribution to a Registered Retirement Savings Plan (RRSP)"},
       {value: "/login/questions/tfsa", label: "Details of your contribution to a Tax Free Savings Account (TFSA)"},
       {value: "/login/questions/ccb", label: "Details of your Canada Child Benefit (CCB)"},
-      {value: "/login/questions/trillium", label: "Your Ontario Trillium Benefit amount and payment type"}
```

Comment:
Nice! Either as part of this PR or a separate one, let's delete the /trillium question (and URL and schema and tests, etc)